### PR TITLE
Merge 3.4-release into master

### PR DESCRIPTION
**Pull request by joaoalf**
Part of the unfreezing process.

TODO:
- [x] Create a new release and update `herokuish` to use it when this PR get merged

@BRONSOLO I had to resolve a couple of conflicts on this one. Can you review, please?

### CODE REVIEW

File: bin/steps/sqlite3 [hunk 1]
```diff
@@ -66,14 +66,21 @@ buildpack_sqlite3_install() {
...
+        echo "Sqlite3 successfully installed."
+        mcount "success.python.sqlite3"
+      else
+        echo "Sqlite3 failed to install."
+        mcount "failure.python.sqlite3"
+      fi
+
+      # save version installed
+      mkdir -p "$CACHE_DIR/.heroku/"
+      echo "$SQLITE3_VERSION" > "$CACHE_DIR/.heroku/python-sqlite3-version"
```
Comment (BRONSOLO) [inline bin/steps/sqlite3]: I would think we would also want to use what is on **master** here as well. This pin was fixed in: https://github.com/heroku/heroku-buildpack-python/commit/ffc7683c05c924d345b43c6ad1c5ea8b4a5a5fb1

File: README.md [hunk 2]
```diff
@@ -56,34 +56,4 @@ to use the new version,
...
-    git checkout -b rebase-on-upstream
-
-    # rebase our commits onto `heroku/heroku-buildpack-python`
-    # resolve any conflicts as necessary
-    # update vendored get-pip.py as necessary
-    git rebase heroku/master
-    git push origin rebase-on-upstream
-
-Once the rebase is complete, simply run through the release process as detailed above.
+- Create a PR, get it approved, and merge.
```
Comment (BRONSOLO) [inline README.md]: We should be using what is on **master** here (new release steps introduced here:https://github.com/plotly/heroku-buildpack-python/commit/1b008b36155e32aceb435b25f88beb6903d36a37)

**Review by BRONSOLO — CHANGES_REQUESTED** [review]
Thanks @joaoalf - the `/bin/steps/python` change looks good! Otherwise, the conflicts should be resolved by using what is on **master** imo.

**Comment by joaoalf** [comment]
@BRONSOLO Updated. Can you please check again?

**Comment by BRONSOLO** [comment]
:dancer: Thanks for the changes - looks good to me @joaoalf.

**Pull request merged**